runtime/js/40_testing.js

```diff
@@ -1013,32 +1205,31 @@
       createTestFilter(filter),
     );
 
+    let groups = new Set([undefined]);
```

Comment:
Why is undefined added here? I think it will be added if needed on line 1213?

Reply:
undefined makes sure that ungrouped benchmarks are placed first in sorted array